components/admin_console/multiselect_settings.jsx

```diff
@@ -57,6 +57,18 @@ export default class MultiSelectSetting extends React.Component {
         }
     }
 
+    calculateValue = () => {
+        return this.props.selected.reduce((values, item) => {
```

Comment:
Just to confirm what this is doing `this.props.selected` is an array of language names like `english,french,spanish`, and this is converting that into a list of objects representing each language? Unless invalid entries from the config.json are going to cause a problem, perhaps we could use map instead of reduce here? 0/5

Reply:
That's correct, although the values are `en`, `fr`, `es` rather than language *names*.

I'd prefer this way as the UI ends up looking broken if we have an unexpected value (not necessarily invalid, just not expected by the UI). Principle of least surprises to me is to not have javascript errors throwing where an unexpected value is conceivable.